code/modules/mob/living/carbon/human/inventory.dm

```diff
@@ -215,15 +215,13 @@ This saves us from having to call add_fingerprint() any time something is put in
 		if(slot_back)
 			src.back = W
 			W.equipped(src, slot)
-			worn_clothing += back

```

Comment:
I'm assuming this is handled by equipped, then?

Reply:
Drunk ater: "What was sober ater thinking, assuming that these additions (Additions!) where actually handled correctly (as ORs) somewhere else?!"

Reply:
I have.. no idea what that means in reference